### PR TITLE
docs(get-started): show correct docker volume path for different platforms

### PR DESCRIPTION
**Pull request by argshook**
This PR adds tabs to docker run command on `/docs/get-started/docker`
page

### CODE REVIEW

File: docs/get-started/docker.md [hunk 1]
```diff
@@ -26,16 +26,44 @@ Once Docker is installed, you will need to pull QuestDB's image from
 
 This can be done with a single command using:
 
-<InterpolateReleaseData
-  renderText={(release) => (
-    <CodeBlock className="language-shell">
-      {`docker run \\
+export const platforms = [
+  { label: "Linux", value: "linux", volumePath: "/var/lib/questdb" },
```
Comment (amyshwang) [inline docs/get-started/docker.md:30]:
```suggestion
  { label: "Linux", value: "linux", volumePath: "$HOME/.questdb" },
```

File: docs/get-started/docker.md [hunk 2]
```diff
@@ -26,16 +26,44 @@ Once Docker is installed, you will need to pull QuestDB's image from
...
+  { label: "Mac with M1/M2", value: "mac-arm", volumePath: "/var/lib/questdb" },
```
Comment (amyshwang) [inline docs/get-started/docker.md:31]:
```suggestion
  { label: "Mac with M1/M2", value: "mac-arm", volumePath: "/opt/homebrew/var/questdb" },
```

File: docs/get-started/docker.md [hunk 3]
```diff
@@ -26,16 +26,44 @@ Once Docker is installed, you will need to pull QuestDB's image from
...
+  {
+    label: "Mac with Intel",
+    value: "mac-intel",
+    volumePath: "/opt/homebrew/var/questdb",
```
Comment (amyshwang) [inline docs/get-started/docker.md:35]:
```suggestion
    volumePath: "/usr/local/var/questdb",
```

File: docs/get-started/docker.md [hunk 4]
```diff
@@ -26,16 +26,44 @@ Once Docker is installed, you will need to pull QuestDB's image from
...
+  },
+  {
+    label: "Windows",
+    value: "windows",
+    volumePath: "\\Windows\\System32\\qdbroot",
```
Comment (amyshwang) [inline docs/get-started/docker.md:40]: This one I am less sure about as your version might be for Docker command?

```suggestion
    volumePath: "\Windows\System32\qdbroot",
```

**Review by amyshwang — CHANGES_REQUESTED** [review]
Correcting root dir path

File: docs/get-started/docker.md (hunk 4 above)
Comment (argshook) [reply docs/get-started/docker.md:40]: a backslash is a special escape character used to indicate other special characters such as new lines (\n), tabs (\t), or quotation marks (\\"). In order to display a backslash `\` we must escape itself. That's why i used two: `\\`

File: docs/get-started/docker.md (hunk 1 above)
Comment (argshook) [reply docs/get-started/docker.md:30]: is this suggestion correct? in other docs (e.g. [here](https://questdb.io/docs/reference/configuration/#docker)) the path suggested is `/var/lib/questdb/`

Comment (amyshwang) [reply docs/get-started/docker.md:30]: Yes - it was used here - https://questdb.io/docs/concept/root-directory-structure/
And I asked @puzpuzpuz to confirm and approve this PR - https://github.com/questdb/questdb.io/pull/1511

**Comment by amyshwang** [comment]
@argshook Apologies - I realise that the path should remain the same because it's the path running within the container!
I will close the ticket. Sorry for the trouble caused!!

File: docs/get-started/docker.md (hunk 1 above)
Comment (puzpuzpuz) [reply docs/get-started/docker.md:30]: My bad. I haven't spotted the elephant in the room - Docker. On Docker our root dir is `/var/lib/questdb` (unless it's an old deployment where we were running under `root` - then it's `/root/.questdb`).

@amyshwang my apologies. I'll submit a fix PR.

**Pull request closed without merging**